loopchain/consensus/syncer.py

```diff
@@ -0,0 +1,160 @@
+import asyncio
+
+import time
+from lft.consensus.events import ReceiveDataEvent, ReceiveVoteEvent
+from typing import TYPE_CHECKING, Dict
+
+from loopchain import configure as conf
+from loopchain.blockchain.blocks.v1_0 import Block
+from loopchain.blockchain.votes.v1_0 import BlockVote
+from loopchain.channel.channel_property import ChannelProperty
+from loopchain.protos import loopchain_pb2, loopchain_pb2_grpc
+from loopchain.tools.grpc_helper import GRPCHelper
+
+if TYPE_CHECKING:
+    from loopchain.peer.block_manager import BlockManager
+    from lft.event import EventSystem
+
+RAISE_EVENT_INTERVAL = 3  # integer
+NONE_VOTE_HEIGHT = 0  # integer
```

Comment:
If you want to guide the type of those variables, I think it is better to be: `VARIABLE: int = 3` without comment.